packages/blocks/src/_common/configs/text-format/config.ts

```diff
@@ -30,87 +25,95 @@ export const textFormatConfigs: TextFormatConfig[] = [
     name: 'Bold',
     icon: BoldIcon,
     hotkey: 'Mod-b',
-    activeWhen: host => commonActiveWhen(host, 'bold'),
+    activeWhen: host => {
+      const [result] = host.std.command
+        .pipe()
+        .isTextStyleActive({ key: 'bold' })
+        .run();
+      return result;
+    },
     action: host => {
-      handleCommonStyle(host, 'bold');
+      host.std.command.pipe().toggleBold().run();
```

Comment:
Could we rename the `pipe` naming to `chain`? Or else which naming looks more intuitive for you?

Reply:
> Could we rename the `pipe` naming to `chain`? Or else which naming looks more intuitive for you?

I think we should rename the infra apis in other pr. And both `chain` and `pipe` look good to me.